db/migrate/20190328124551_add_prosecution_evidence_to_claim.rb

```diff
@@ -0,0 +1,5 @@
+class AddProsecutionEvidenceToClaim < ActiveRecord::Migration[5.2]
+  def change
+    add_column :claims, :prosecution_evidence, :boolean, default: nil
```

Comment:
how is the nil default playing-out on the view? as we discussed yesterday

Reply:
It manifests as a pair of unselected radio buttons.